dnf/automatic/emitter.py

```diff
@@ -130,7 +131,8 @@ def commit(self):
         stdin_feed = stdin_fmt.format(**msg).encode('utf-8')
 
         # Execute the command
-        subp = subprocess.Popen(command, shell=True, stdin=subprocess.PIPE)
+        args = shlex.split(command)
```

Comment:
Why you split arguments and still use shell=True?

Reply:
Hi, sorry I missed your comment. I did only minimal changes to fix my problems with `command_email`. Should the `shell=True` be removed?

Reply:
@j-mracek I have tested it without `shell=True" and I receive `OSError: [Errno 36] File name too long:`. With `shell=True` it works as expected.